Connection.php

```diff
@@ -67,6 +82,67 @@ public function __construct($config = [])
         }
     }
 
+    private $multiExecCommand   = false;
+    private $multiCommands      = [];
+    private $redisRetryCommands = [
+        'retrySet'      => 'SET',
+        'retryGet'      => 'GET',
+        'retryKeys'     => 'KEYS',
+        'retryDel'      => 'DEL',
+        'retryTTL'      => 'TTL',
+        'retryMulti'    => 'MULTI',
+        'retrySAdd'     => 'SADD',
+        'retryExec'     => 'EXEC',
+        'retryExpire'   => 'EXPIRE',
+        'retrySMembers' => 'sMembers',
+        'retryUnlink'   => 'UNLINK',
+        'retryPublish'  => 'PUBLISH',
+    ];
+
+    /**
+     * Allows issuing all supported commands via magic methods.
+     *
+     * @param string $command - name of the missing method to execute
+     * @param array  $params  - method call arguments
+     * 
+     * @return mixed
+     */
+    public function __call($command, $params)
+    {
+        if (in_array($command, array_keys($this->redisRetryCommands))) {
+            if ($command === 'retryMulti') {
+                $this->multiExecCommand = true;
+                $this->multiCommands[]  = $this->computeRawCommand($command, $params);
```

Comment:
**question:** where do we ever reset or clear the `multiCommands` array?

Reply:
@peterjjohnson Good catch! I forgot to add that cleaning. I added it now in 2 locations for safely: when retryMulti is sent, before starting computing the raw commands and after the retryExec is executed to clean it.